src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -149,6 +177,34 @@ public class NewAzureContainerGroupCommand : ContainerInstanceCmdletBase
         [ValidateNotNullOrEmpty]
         public PSCredential RegistryCredential { get; set; }
 
+        [Parameter(
```

Comment:
Quick recommendation: move these parameters above all of the non-mandatory parameters so that the users will see them more quickly when scanning help.

Reply:
But these are only mandatory for this parameter set. Should I still do the same?

Reply:
Yes, they are not in the other parameter sets, so it won't affect those other parameter sets.